app/src/marketplace/containers/ProductController/useValidationCallback.js

```diff
@@ -0,0 +1,20 @@
+// @flow
+
+import { useCallback } from 'react'
+
+import type { Product } from '$mp/flowtype/product-types'
+import useValidation, { ERROR, OK } from './useValidation'
+
+export default function useValidationCallback() {
```

Comment:
I wonder if this should just be `useProductValidationEffect` instead. Though separating along the line of `useProduct` also makes sense.